hibernate-core/src/main/java/org/hibernate/boot/MetadataBuilder.java

```diff
@@ -225,6 +225,23 @@
 	MetadataBuilder enableNewIdentifierGeneratorSupport(boolean enable);
 
 	/**
+	 * Should we enable support for the "new" (since 5.2) type conflict resolving.
+	 *
+	 * Its default is defined by the {@link org.hibernate.cfg.AvailableSettings#USE_NEW_TYPE_CONFLICT_RESOLVER}
+	 * setting if using property-based configuration.
+	 *
+	 *
+	 * @param enable {@code true} to enable; {@code false} to disable;don't call for
+	 * default.
+	 *
+	 * @return {@code this}, for method chaining
+	 * @since 5.2.9
```

Comment:
I realize this is nit-picking, but be mindful of using `@since` declarations in bug fixes.  This is typically fine to add to new features because those are rarely back-ported, but bug fixes often may and this just creates unnecessary overhead when we do.